crates/runtime/benches/bench_spicecloud/mod.rs

```diff
@@ -143,11 +143,13 @@ fn get_tpch_test_queries() -> Vec<(&'static str, &'static str)> {
         ("tpch_q14", include_str!("../queries/tpch/q14.sql")),
         // tpch_q15 has a view creation which we don't support by design
         ("tpch_q16", include_str!("../queries/tpch/q16.sql")),
-        ("tpch_q17", include_str!("../queries/tpch/q17.sql")),
+        // DoGet recv error: rpc error: code = InvalidArgument desc = Correlated scalar subquery can only be used in Projection, Filter, Aggregate plan nodes
```

Comment:
Why do these queries fail?

Reply:
I'm tracking this in a separate issue https://github.com/spiceai/spiceai/issues/3980

q2 and q17 are failing with 
```
DoGet recv error: rpc error: code = InvalidArgument desc = Correlated scalar subquery can only be used in Projection, Filter, Aggregate plan nodes
```

q20
```
DoGet recv error: rpc error: code = InvalidArgument desc = No field named l1.l_orderkey. Valid fields are __correlated_sq_1.l_orderkey, __correlated_sq_1.l_suppkey.
```

simple_q3
```
Tonic error: status: ResourceExhausted, message: "DoGet recv error: rpc error: code = ResourceExhausted desc = grpc: received message larger than max (1646208435 vs. 104857600)", details: [], metadata: MetadataMap { headers: {} }
```

I think all errors are from flight client, which we use for spice.ai